src/openrct2-ui/windows/LandRights.cpp

```diff
@@ -308,11 +308,11 @@ namespace OpenRCT2::Ui::Windows
             }
 
             // Position land size tool
-            widgets[WIDX_PREVIEW].top = 17;
+            widgets[WIDX_PREVIEW].top = 17 + getTitleBarDiffNormal();
```

Comment:
Why not make this relative to the page widget, like I did for all other cases I encountered?

Reply:
Yeah, that might be nicer. Let me change it to that then.